roles/rciam-probes/vars/CentOS-8.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Should this be renamed to RedHat-8.yml for consistency with RedHat.yml in other roles?

Reply:
Done